db/migrations/versions/1147a99f5cc8_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 1147a99f5cc8
+Revises: a0b9e8f859a9
+Create Date: 2022-11-02 14:44:32.698940
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "1147a99f5cc8"
+down_revision = "a0b9e8f859a9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("applications", schema=None) as batch_op:
+        batch_op.alter_column(
```

Comment:
Am I right in understanding this is just changing the type from `String` to `VARCHAR` if so can we delete this migration and just update the first one?

Reply:
Makes sense to me, is there any another reason this is split out @gio-karunakaran? 👀

Reply:
I made language an Enum and got rid of the two migrations in place of one

Reply:
that makes sense to me! 👍🏾